woocommerce/woocommerce-android
# Fix position changes after update

Part of: #9541

### Description
This small PR fixes the bundle items from switching positions after updating the item's quantities. It does that by sorting the items by ID before displaying them in the UI.

### Testing instructions
 1. Open the orders tab
 2. Create a new order
 3. Add a product bundle (including a valid configuration)
 4. Add a different product bundle (including a valid configuration)
 5. Select the products
 6. Check that the products are added to the order
 7. Increase / Decrease the product quantity
 8. Check that the products are updated accordingly
 9. Check that the items don't switch positions after the update

### Images/gif

#### Before
https://github.com/woocommerce/woocommerce-android/assets/18119390/f4e2ce30-05da-4d7b-a506-c04ab1739412

#### After
https://github.com/woocommerce/woocommerce-android/assets/18119390/407228c1-80e2-45bc-a935-97bcc74282a3
